packages/engine-core/src/data-proxy/utils/getClientVersion.ts

```diff
@@ -1,22 +1,34 @@
 import type { EngineConfig } from '../../common/Engine'
+import { NotImplementedYetError } from '../errors/NotImplementedYetError'
 
 /**
  * Determine the client version to be sent to the DataProxy
  * @param config
  * @returns
  */
-export function getClientVersion(config: EngineConfig) {
-  const [version, suffix] = config.clientVersion?.split('-') ?? []
-
-  // we expect the version to match the pattern major.minor.patch
-  if (!suffix && /^[1-9][0-9]*\.[0-9]+\.[0-9]+$/.test(version)) {
-    return version
+export function getClientVersion(config: EngineConfig): string {
+  if (process.env.PRISMA_CLIENT_DATA_PROXY_CLIENT_VERSION) {
+    // should we log with debug here that this env var is used?
```

Comment:
Yes, that sounds like a good idea to make this clearly visible.

Reply:
Done